interop/client/client.go

```diff
@@ -214,6 +250,22 @@ func main() {
 	if len(*serviceConfigJSON) > 0 {
 		opts = append(opts, grpc.WithDisableServiceConfig(), grpc.WithDefaultServiceConfig(*serviceConfigJSON))
 	}
+	addMd, err := parseAdditionalMetadataFlag()
+	if err != nil {
+		logger.Fatalf("Error parsing --additional_metadata flag: %v", err)
```

Comment:
Optional: IMO just put the `logger.Fatalf` inside the function and have it not return an error.  But it's fine if you don't want to change it.

Reply:
done